### DISCUSSION TITLE
Shelly 3em pro mit einer Phase betreiben

### DISCUSSION THREAD

**Discussion by Den176** [matlen67/soyosource-powercontroller#15] [General]
Hallo Matlen67,

Mit dem aktuellen Update bei Shelly kann man den 3em pro auch auf eine Phase umstellen.
Somit hat man dann drei voneinander unabhängige Energiemeter.

![Screenshot_20241012_232349_Shelly Smart Control](https://github.com/user-attachments/assets/02bb0be8-2098-4cb5-b544-f5c72c966e9e)

Das ist von großem Vorteil wenn das Haus ohnehin nur an einer Phase angeschlossen ist was bei älteren Gebäuden öfters der Fall ist.

Deine Software erkennt den 3em pro dann zwar noch als solchen aber der Json Pfad hat sich geändert und somit findet das Programm keinen Messwert.

Wäre es denn möglich das du diesen Json in deiner Software noch mit aufnimmst?

GET Url bleibt gleich: http://ip/rpc/Shelly.GetStatus

power 1 =  doc ["em1:0"] ["act_power"];
power 2 =  doc ["em1:1"] ["act_power"];
power 3 =  doc ["em1:2"] ["act_power"];

![Screenshot_20241012_232703_Chrome](https://github.com/user-attachments/assets/db70cbcd-dd13-420a-a0bf-70e373718abf)

Vielen Dank!

Grüße 
Dennis

**Comment by matlen67** [comment]
Moin, habe ich mit eingebaut. Bitte mal testen da ich meinen 3EM Pro nicht umstellen möchte

**Comment by Den176** [comment]
Funktioniert! 👍
Super, vielen Dank für das schnelle einbauen! 😀
Leider hab ich fürs selber ändern zu wenig Ahnung vom programmieren.......
